xil_cloud_telematics/docker-compose-background.yml

```diff
@@ -1,29 +0,0 @@
-#  Copyright (C) 2018-2021 LEIDOS.
```

Comment:
This file doesn't seem to have existed in the xil_cloud config, was the xil_cloud_telematics config started from a different template?